docs/index.md

```diff
@@ -23,7 +23,7 @@ The AsyncAPI generator is a tool that generates anything you want using the **[A
 2. The **Generator** sends to the **[Parser](generator/parser)** the **asyncapiString** which is a stringified version of the original **AsyncAPI Document**.
 3. The **Parser** uses additional plugins such as the OpenAPI, RAML, or Avro schemas to validate custom schemas of message payloads defined in the **AsyncAPI Document**.
 4. If the **Parser** determines that the original **AsyncAPI Document** is valid, it manipulates the document and returns a set of helper functions and properties and bundles them together into an **asyncapi** variable that is an instance of [**AsyncAPIDocument**](https://github.com/asyncapi/parser-api/blob/master/docs/api.md#asyncapidocument). The **asyncapi** helper functions make it easier to access the contents of the AsyncAPI Document.
-5. At this point, the **Generator** passes the **[asyncapi](generator/asyncapi-document#method-2-asyncapi-and-template)**, the **[originalAsyncAPI](generator/asyncapi-document#method-1-originalasyncapi-and-template)**, and the **params** which collectively make up the **[Template Context](generator/asyncapi-context)** to the **Render Engine**. 
+5. At this point, the **Generator** passes the **[asyncapi](generator/asyncapi-document#method-2-asyncapi-and-template)**, the **[originalAsyncAPI](generator/asyncapi-document#method-1-originalasyncapi-and-template)**, and the **params** which collectively make up the **[Template Context](https://www.asyncapi.com/docs/tools/generator/template-context)** to the **Render Engine**. 
```

Comment:
@Florence-Njeri I think link should be `generator/template-context` so it is relative to other pages on website - look on other links

thoughts?

Reply:
<img width="1113" alt="Screenshot 2024-04-09 at 8 04 51 PM" src="https://github.com/asyncapi/generator/assets/16447041/e208dca7-0531-460f-973a-8ee1051d9b48">

if I modify the link to be `generator/template-context`, the link cannnot be opened and the IDE gives me this error.

Reply:
yup, that is "expected" because the link we keep in markdown are expected to work in asyncapi.com website and we ignore local/github cause in the end we want people to read docs in asyncapi.com

Reply:
I see! Once @Florence-Njeri responds affirmatively I will make the change : )

Reply:
@lmgyuan I agree with @derberg about the link

Reply:
Just updated the link! @derberg @Florence-Njeri